libs/hmpb/layout/src/layout.ts

```diff
@@ -1683,12 +1683,20 @@ function layOutBallotHelper({
     },
     gridLayout: {
       ballotStyleId: ballotStyle.id,
-      optionBoundsFromTargetMark: {
-        bottom: 1,
-        left: 1,
-        right: 9,
-        top: 1,
-      },
+      optionBoundsFromTargetMark:
+        bubblePosition === 'left'
```

Comment:
I don't mind not having tests for the debug stuff, but this feels like it deserves a test. Unless you consider the changes to the fixtures as sufficient?

Reply:
I've generally been relying on the fixtures to test this. Your question is making me think about what would make for a good test of write-in cropping though. Maybe actually cropping an image of the ballot and snapshotting that so that if it changes, we'd be looking at a visual diff?

Reply:
I do think that'd be a useful test.

Reply:
[089546e](https://github.com/votingworks/vxsuite/pull/4444/commits/089546ef1fffe684872e5faf342e9319961f7d00)